device/ospf/ospfv2/router.yaml

```diff
@@ -0,0 +1,171 @@
+components:
+  schemas:
+    Device.Ospfv2:
+      description: >-
+        A container of properties for an OSPFv2 router and its interfaces & Route Ranges.
+      type: object
+      required: [router_id, interfaces, name]
+      properties:
+        name:
+          x-include: ../common/common.yaml#/components/schemas/Named.Object/properties/name
+          x-field-uid: 1
+        router_id:
+          description: >-
+            IP address of Router ID for this emulated OSPFv2 router.
+          type: string
+          format: ipv4
+          x-field-uid: 2
+        lsa_retransmit_timer:
+          description: >-
+            The time in seconds for LSA retransmission.
+          type: integer
+          format: uint32
+          default: 5
+          minimum: 1
+          x-field-uid: 3
+        lsa_refresh_time:
+          description: >-
+            The time in seconds required for LSA refresh.
+          type: integer
+          format: uint32
+          default: 1800
+          minimum: 5
+          x-field-uid: 4
+        inter_burst_lsu_interval:
+          description: >-
+            The gap in miliseconds between each Flood Link State Update Burst 
+          type: integer
+          format: uint32
+          default: 33
+          x-field-uid: 5
+        max_flood_lsu_per_burst:
+          description: >-
+            The maximum number of Flood LSUpdates for each burst 
+          type: integer
+          format: uint32
+          default: 1
+          minimum: 1
+          x-field-uid: 6
+        graceful_restart:
+          $ref: '#/components/schemas/Ospfv2.GracefulRestart'
+          x-field-uid: 7
+        learned_lsp_filter:
+          description: >-
+            Configuration for controlling storage of OSPFv2 learned LSPs are received
```

Comment:
OSPFv2 should be LSAs

Reply:
done

Reply:
Still see issues. Mentioned in new review . Closing this.